datasets/interpress_news_category_tr/interpress_news_category_tr.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
+# Lint as: python3
+"""Turkish News Category Dataset (270K) - Interpress Media Monitoring Company"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import logging
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+It is a Turkish news data set consisting of 273601 news in 17 categories, compiled from print media and news websites between 2010 and 2017 by the Interpress (https://www.interpress.com/) media monitoring company.
+"""
+
+_CITATION = ""
+
+_HOMEPAGE = "https://www.interpress.com/"
+_DOWNLOAD_URL = "https://www.interpress.com/downloads/interpress_news_category_tr_270k.zip"
+_DATASET_URLS = {
+    "train": "interpress_news_category_tr_270k_train.tsv",
+    "test": "interpress_news_category_tr_270k_test.tsv",
+}
+
+
+class InterpressNewsCategoryTRConfig(datasets.BuilderConfig):
+    """BuilderConfig for InterpressNewsCategoryTR."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for InterpressNewsCategoryTR.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(InterpressNewsCategoryTRConfig, self).__init__(**kwargs)
+
+
+class InterpressNewsCategoryTR(datasets.GeneratorBasedBuilder):
+    """Turkish News Category Dataset (270K) - Interpress Media Monitoring Company"""
+
+    BUILDER_CONFIGS = [
+        InterpressNewsCategoryTRConfig(
+            name="interpress_news_category_tr_270k",
```

Comment:
Might be a tad long as a config name 😄 
May be just "270k" ? So that one can load the dataset with the following command:
`data = load_dataset("interpress_news_category_tr", "270k")`

Reply:
We didn't know we could use it this way. Thank you. We update it this way.